packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -573,31 +577,17 @@ export class RunningSummarizer implements IDisposable {
 				this.beforeSummaryAction();
 			},
 			async () => {
-				const attempts: (ISummarizeOptions & { delaySeconds?: number })[] = [
+				if (this.mc.config.getBoolean("Fluid.Summarizer.TryDynamicRetries")) {
+					return this.trySummarizeWithDynamicRetries(reason, cancellationToken);
+				}
+
+				const attempts: ISummarizeOptions[] = [
 					{ refreshLatestAck: false, fullTree: false },
 					{ refreshLatestAck: true, fullTree: false },
-					{ refreshLatestAck: true, fullTree: false, delaySeconds: 2 * 60 },
-					{ refreshLatestAck: true, fullTree: true, delaySeconds: 10 * 60 },
 				];
-				let overrideDelaySeconds: number | undefined;
 				let summaryAttempts = 0;
 				let summaryAttemptsPerPhase = 0;
-				// Reducing the default number of attempts to defaultNumberofSummarizationAttempts.
-				let totalAttempts =
-					this.mc.config.getNumber("Fluid.Summarizer.Attempts") ??
-					defaultNumberSummarizationAttempts;
-
-				if (totalAttempts > attempts.length) {
-					this.mc.logger.sendTelemetryEvent({
-						eventName: "InvalidSummarizerAttempts",
-						attempts: totalAttempts,
-					});
-					totalAttempts = defaultNumberSummarizationAttempts;
-				} else if (totalAttempts < 1) {
-					throw new UsageError("Invalid number of attempts.");
-				}
-
-				for (let summaryAttemptPhase = 0; summaryAttemptPhase < totalAttempts; ) {
+				for (let summaryAttemptPhase = 0; summaryAttemptPhase < attempts.length; ) {
```

Comment:
side note: let's make this a while loop to make it clear it doesn't always increment

Reply:
I reverted most of the changes here. Please take a look.